docs/gateway/b2.md

```diff
@@ -52,6 +51,7 @@ Gateway inherits the following B2 limitations:
 Other limitations:
 
 - Bucket notification APIs are not supported.
+- The PutObject S3 API only returns the md5sum of an uploaded file as it's etag response when minio is started in --compat mode. Use this only if you are stuck with a client library where this check cannot be turned off (e.g. boto).
```

Comment:
can you show on example where this cannot be turned off @KiNgMaR ? -

Reply:
Sure!
- for boto 2: https://github.com/boto/boto/blob/9e1cd3bd76e738d80630f1bd9160fd87c8eab865/boto/s3/key.py#L1000
- fot boto 3: https://github.com/boto/boto3/issues/544